src/nncf/torch/model_graph_manager.py

```diff
@@ -371,6 +372,43 @@ def get_weight_channel_axes(metatype: type[OperatorMetatype], ndims: int, input_
     return (0,)
 
 
+def get_reduction_axes_from_metatype(metatype: OperatorMetatype, weight_port_id: int, ndims: int):
```

Comment:
Looks like this function is never used. Please remove previous implementation and use this function

Reply:
Is it possible to reuse this function in the `torch_backend.py`?